dir.traversal.targets

```diff
@@ -86,6 +86,32 @@
              ContinueOnError="ErrorAndContinue" />
   </Target>
 
+  <Target Name="ProducesPackageId"
+          Returns="@(PackageIds)" 
```

Comment:
For both of the scenarios I know, packages will be built anyway:
 1. Doing an offline product build. Call `produces` on corefx, call `consumes` on repo B that depends on corefx, change the `consumes` output, pass the modified output into B's `change`.
 2. My current plan for auto-upgrade. Same flow, but passing `produces` output "through" the versions repo.

Now I'm wondering about the runtime packages, for (1)... reading the repo API docs, it seems like there's the assumption that a repo always produces the same artifacts, which isn't the case for corefx (or any repo with native compilation, if docker and cross-compilation aren't involved). Should `produces` be scoped to the full set of possible packages, the ones that would be built with a default "build" on the current machine, the packages that are built no matter the platform, or something else?